server/auth_function/lambda_function.py

```diff
@@ -82,17 +82,23 @@ def lambda_handler(event: event_type.Event, context: Any) -> event_type.Event:
         audit_event_log["requestingUser"]["userType"] = USER_TYPE_CODE_DICT[
             event["request"]["userAttributes"]["custom:idp_name"]
         ]
-        if (
-            audit_event_log["requestingUser"]["userType"] == USER_TYPE_IDIR
-            or audit_event_log["requestingUser"]["userType"] == USER_TYPE_BCEID_BUSINESS
-        ):
+
+        if audit_event_log["requestingUser"]["userType"] == USER_TYPE_IDIR:
+            audit_event_log["requestingUser"]["idpUserName"] = event["request"][
+                "userAttributes"
+            ]["custom:idp_username"]
+        elif audit_event_log["requestingUser"]["userType"] == USER_TYPE_BCEID_BUSINESS:
             audit_event_log["requestingUser"]["idpUserName"] = event["request"][
                 "userAttributes"
             ]["custom:idp_username"]
+            audit_event_log["requestingUser"]["businessGuid"] = event["request"][
+                "userAttributes"
+            ]["custom:idp_business_id"]
```

Comment:
added the business guid to the audit log if login user's user type is business bceid